gaia-header.js

```diff
@@ -1,9 +1,10 @@
+/* jshint node:true */
+/* globals define */
```

Comment:
I do agree it's sensible that Gaia related projects share similar coding conventions, but it shouldn't be Gaia's job to enforce these conventions in its own build.